### DISCUSSION TITLE
Flash Floppy for Korg Trinity

### DISCUSSION THREAD

**Discussion by w2helm** [keirf/flashfloppy#882] [General]
I have found a real paucity of information on using FF with Korg Trinity.  In fact I haven't read any success stories.  So attached is a detailed one.  Yes, it works.  It works correctly.  I have even successfully used it to upgrade my Trinity Pro X (V2) to Version 2.4.1.  If you are looking for a floppy disk replacement for a Trinity, this is the article you have been looking for.  It covers all the details thoroughly.  I had never used a Trinity and never used a GoTek or Flash Floppy, so I documented what I learned.
[Floppy Emulator.docx](https://github.com/keirf/flashfloppy/files/14671857/Floppy.Emulator.docx)
[V241images.zip](https://github.com/keirf/flashfloppy/files/14671860/V241images.zip)

**Comment by ParietalLab** [comment]
Thanks for sharing your experience. I went through many iterations to finally get this working on a Korg Trinity also. The missing piece for me was setting pin34 = auto in my FF.CFG file. I'm sharing this here in case others continue to be stuck.

FF.CFG (placed either at root of USB drive or in FF folder of drive)
host = akai
interface = shugart
pin02 = auto
pin34 = chg
write-protect = no
image-on-startup = last

**Reply by Wizards-of-Zen** [reply]
you need Pins S1 and JC jumpered on.

the only format you need on a thumb drive is a FAT32 file system. 

use Flash Floppy file name format DSK0000.img - DSK0xxx.img

used winimage to inject korg files into the .img files

Does anyone have a v3.12 os file set as disk images or just what as on the floppies my trinity is still on 3.0.0, i'll have to find a pc with a floppy disk to make those...
